target/linux/ath79/dts/ar9341_engenius_eap300-v2.dts

```diff
@@ -12,6 +12,7 @@
 	compatible = "engenius,eap300-v2", "qca,ar9341";
 
 	aliases {
+		label-mac-device = &wmac;
```

Comment:
We are using this [patch](https://git.openwrt.org/?p=openwrt/openwrt.git;a=blob;f=target/linux/generic/pending-6.6/683-of_net-add-mac-address-to-of-tree.patch;hb=70610a5240a94648e178713f091435bd0ba158d3) to add the label-mac entry. So you need to assign mac address via nvmem cell for `wmac` dts node to make it work.

Reply:
Interesting. A bunch of devices need fixing looks like.

Reply:
```
121f
pineapple-nano
ar9342_ubnt_xw.dtsi
n5q
carambola3
hh40v
r36a
qca9533_ubnt_aircube-isp
qca9533_yuncore_cpe830.dts
mt7620a_alfa-network_r36m-e4g.dts
mt7620a_glinet_gl-mt300a.dts
mt7620a_glinet_gl-mt300n.dts
mt7620a_glinet_gl-mt750.dts
mt7620a_zbtlink_zbt-we1026.dtsi
mt7628an_cudy_wr1000.dts
mt7628an_vocore_vocore2.dtsi
mt7628an_xiaomi_mi-router-4.dtsi
mt7628an_zbtlink_zbt-we2426-b.dts
rt3050_dlink_dir-615-d.dts
rt3352_dlink_dir-615-h1.dts
```

That's everything in ramips and some in ath79.